docs/platform/howto/manage-vpc-peering.rst

```diff
@@ -30,6 +30,11 @@ When you create a new service, you can then place it in the VPC. The **VPC** tab
    .. note::
        Depending on the cloud provider that you selected for the VPC connection, you also have to accept a VPC peering connection request or set up a corresponding VPC peering connection to Aiven. 
 
+Cloud-specific VPC peering instructions
+-----------------------------------------------------
+
+- Refer to :ref:`GCP VPC peering instructions <vpc-peering-gcp>`.
```

Comment:
```suggestion
- :ref:`Set up VPC peering on Google Cloud Platform (GCP) <vpc-peering-gcp>`.
```

Reply:
Small suggestion to spell out GCP and make the language more consistent with the page the user is navigating to.